modules/bitgo/test/v2/fixtures/lightning/invoices.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+  lnurlPayInvoice: {
+    comment: 'taken from https://github.com/lightning/bolts/blob/master/11-payment-encoding.md',
```

Comment:
typescript file is OK as well, you can use `//` for comment now

Reply:
I tried to use json file, but it gave me `include` error saying all files should be included. I checked `tsconfig.json` , and it seemed everything is included.